src/style.css

```diff
@@ -2,2 +1,83 @@
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.container-logo {
+  display: flex;
+}
+
+.container-logo img {
+  display: block;
+}
+
+.nav {
+  flex-grow: 1;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.nav__list {
+  display: flex;
```

Comment:
You are using the tag name `ul` to style the navigation list. It's better to style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. In this case, you have the class `.nav__list` which is great.